sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -180,6 +180,7 @@ public static Iterable<Object[]> data() {
         new Object[] {false, false, false},
         new Object[] {false, false, true},
         new Object[] {true, false, false},
+        new Object[] {true, true, false},
```

Comment:
I noticed storage write at least once mode for batch wasn't included in these tests.